requirements_test.txt

```diff
@@ -11,3 +11,4 @@ pytest-xdist~=2.4
 # Type packages for mypy
 types-pkg_resources==0.1.3
 types-toml==0.10.0
+gitpython>3
```

Comment:
This will need to be added to `requirements_tests_min` I think.

At least Windows tests seem to use that file for their requirements:
https://github.com/PyCQA/pylint/blob/62758ae7026398f556bcde2215fc4051b1764ef5/.github/workflows/ci.yaml#L363